packages/components/src/Checkbox/Checkbox.css

```diff
@@ -1,12 +1,11 @@
-.checkbox {
-  --checkbox-size: var(--space-base);
+.checkBoxParent {
```

Comment:
Renamed this class because reading things like `.checkbox .checkBox` was seriously hard to parse